src/agent/Makefile

```diff
@@ -281,7 +292,9 @@ target_version_txt: internals_jar
 #    (Identical to #2, retained for legacy purposes).
```

Comment:
The comment above this is stale and should be updated.

Reply:
Done.